build/source/netcdf/def_output.f90

```diff
@@ -212,7 +212,8 @@ subroutine ini_create(nGRU,nHRU,nSoil,infile,ncid,err,message)
 
  ! create output file
  !err = nf90_create(trim(infile),NF90_64BIT_OFFSET,ncid)
- err = nf90_create(trim(infile),NF90_HDF5,ncid)
+ !err = nf90_create(trim(infile),NF90_HDF5,ncid)
+ err = nf90_create(trim(infile),NF90_NETCDF4,ncid)
```

Comment:
why NETCDF4? Another PR?

Reply:
This is worth discussing again, perhaps.  There was a long interaction on it in AIST because NASA was having trouble with the NF90_HDF5, which is apparently also now a deprecated mode.  The NF90_64BIT_OFFSET was motivated to handle large files, but I think does not handle int64s.  Naoki also discussed with Eric K and his advice was to stick with NETCDF4 as being more solid.  If we're offering a default, this seemed to be the one.  I've been using this with all the runs since the NASA discussions.  Others probably know more than I but I didn't want this to get lost.  Perhaps raising the other two modes as options in a brief code comment is a way to give a safe option and let users 'upgrade' if their needs require it.

Reply:
OK, that would be good. Would you want to add a separate PR for this so (as you say) it does not get lost?

Reply:
sure, I can separate them.